.github/workflows/workflow-pull_request.yml

```diff
@@ -4,6 +4,11 @@ on:
   pull_request:
     branches:
       - master
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
```

Comment:
is this supposed to filter out 'draft'?
- is this list complete then, and not missing any?
- isn't this duplicate with the check on draft down below?

Reply:
@jannisvisser This change is supposed to run the workflow when the _status changes_ from draft to ready for review.
The other check is to skip the job if the _status is_ draft.

We have to check if this works as intended which is only possible across PRs on GitHub.

I got this solution from https://github.com/reviewdog/action-eslint/issues/29#issuecomment-985939887